test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationTest.cs

```diff
@@ -122,61 +115,37 @@ public Task Localization_CustomCulture()
-        [OSSkipCondition(OperatingSystems.Windows)]
-        [FrameworkSkipCondition(RuntimeFrameworks.CoreCLR)]
-        public Task Localization_ResourcesInFolder_ReturnLocalizedValue_WithCultureFallback_Clr()
-        {
-            var testRunner = new TestRunner(_applicationPath);
-            return testRunner.RunTestAndVerifyResponse(
-                RuntimeFlavor.Clr,
-                RuntimeArchitecture.x64,
-                "http://localhost:5072",
-                "ResourcesInFolder",
-                "fr-FR-test",
-                "Bonjour from StartupResourcesInFolder Bonjour from Test in resources folder Bonjour from Customer in resources folder Bonjour from Test in resources folder");
-        }
-
-        [ConditionalFact]
-        [OSSkipCondition(OperatingSystems.Windows)]
-        [FrameworkSkipCondition(RuntimeFrameworks.Mono)]
-        public Task Localization_ResourcesInFolder_ReturnLocalizedValue_CoreCLR_NonWindows()
+        [ConditionalTheory]
+        [OSSkipCondition(OperatingSystems.Linux)]
+        [OSSkipCondition(OperatingSystems.MacOSX)]
+        [InlineData(RuntimeFlavor.Clr, "http://localhost:5070/", RuntimeArchitecture.x64)]
+        public Task Localization_ResourcesInFolder_ReturnLocalizedValue_Windows(
+            RuntimeFlavor runtimeFlavor,
+            string applicationBaseUrl,
+            RuntimeArchitecture runtimeArchitecture)
         {
             var testRunner = new TestRunner(_applicationPath);
             return testRunner.RunTestAndVerifyResponse(
-                RuntimeFlavor.CoreClr,
-                RuntimeArchitecture.x64,
-                "http://localhost:5073/",
+                runtimeFlavor,
+                runtimeArchitecture,
+                applicationBaseUrl,
                 "ResourcesInFolder",
                 "fr-FR",
                 "Bonjour from StartupResourcesInFolder Bonjour from Test in resources folder Bonjour from Customer in resources folder Bonjour from Test in resources folder");
         }
 
-        [ConditionalFact]
-        [OSSkipCondition(OperatingSystems.Windows)]
-        [FrameworkSkipCondition(RuntimeFrameworks.Mono)]
-        public Task Localization_ResourcesInFolder_ReturnLocalizedValue_WithCultureFallback_CoreCLR_NonWindows()
+        [Theory]
+        [InlineData(RuntimeFlavor.CoreClr, "http://localhost:5072", RuntimeArchitecture.x64)]
+        public Task Localization_ResourcesInFolder_ReturnLocalizedValue_WithCultureFallback_AllOS(
+            RuntimeFlavor runtimeFlavor,
+            string applicationBaseUrl,
+            RuntimeArchitecture runtimeArchitecture)
         {
             var testRunner = new TestRunner(_applicationPath);
             return testRunner.RunTestAndVerifyResponse(
-                RuntimeFlavor.CoreClr,
-                RuntimeArchitecture.x64,
-                "http://localhost:5073/",
+                runtimeFlavor,
+                runtimeArchitecture,
+                applicationBaseUrl,
```

Comment:
Here's one of the `[ConditionalFact]` -> `[Fact]` conversions that went strange. Just do the simple thing.